### ISSUE TITLE
Add automated saving of weekly ward progress

### ISSUE THREAD

**Issue by DForbush** [Chicago/census2020_ward_rpt#3]
In the R markdown file, we should think of a way to automatically save the response rate week-to-week. It should be done in a way that it can be automatically recalled by the code week to week, i.e. so that it can be used to create something like a graph of the response rate over time. This will likely involve saving an .Rda file that is automatically recalled and re-saved week to week.

**Comment by sherryshenker** [comment]
hmm we are planning to maintain a table in Platform that has the tract's response rate for every day that we receive data, and plotting that over time in CIC. would using that visualization, and/or the underlying table, solve this?

**Comment by DForbush** [comment]
Yes, that would be perfect. It would be better to have it saved in Civis than in an .Rda file and then call it like any other table through the API.

**Comment by geneorama** [comment]
I wrote a little function to get the daily values from the API, but there are no historical values. 

@sherryshenker thank you for sharing `public.raw_data_2020` which has the history. 

Do you have any idea if they have statistics on how many forms they sent out? I only see daily and cumulative percentages. (I would suggest adding the daily variables to your query... might tell us more? `DRRALL`,`DRRINT`)

I am using this as my query `https://api.census.gov/data/2020/dec/responserate?get=GEO_ID,RESP_DATE,DRRALL,CRRALL,DRRINT,CRRINT&for=tract:*&in=state:17%20county:031&key=APIKEY`

Also, I don't see anything about the return rate or the enumeration rate. Obviously the enumeration hasn't started, but it's disconcerting to me that there is no field. 

Thoughts on that?
